### ISSUE TITLE
double click on a reference will not open target window editor

### ISSUE THREAD

**Issue by djmisterjon** [microsoft/vscode#79633]
hi, if you select a method, right click and select search reference.
Then if you double click in a reference, if the ref are in other window editor, the editor not open `scaling`.
Can we fix this bug plz :)
![image](https://user-images.githubusercontent.com/24865815/63509892-448c2e00-c4ab-11e9-9e07-16cc0a87fa97.png)

**Comment by sbatten** [comment]
@djmisterjon can you check your setting `workbench.useExperimentalGridLayout`

**Comment by djmisterjon** [comment]
it activated but what this should do ?

**Comment by sbatten** [comment]
@djmisterjon It seems that setting does not make a difference for me anyway. I see the collapsed editor group is not expanded when focused this way. Additionally, I see that there is some extra whitespace at the bottom of the image, is that happening in your window?

**Comment by djmisterjon** [comment]
yes it focus on the correct line with highlight, but i want auto `expanded` the window.
And yes you true, it work in debug Mode with breakpoint, when travel in breakpoint it auto open and expanded window needed.
But in this ways, it not work, maybe a dev forget add the feature ?

**Comment by bpasero** [comment]
I think the issue is as follows:
* there is an editor group minimized that is not active
* clicking on the reference will prefer to reveal the editor from the group instead of opening it in the current group
* we preserve focus in the reference view because the interaction is by mouse and as such we do not activate the minimized group
* as such the group does not maximize 

I pushed a change so that a minimized editor group will maximize even if not getting activated.

**Comment by djmisterjon** [comment]
@bpasero yes, sorry, some difficult to explain my brain in english but yes you say all.

**Comment by bpasero** [comment]
@djmisterjon no worries. I am actually thinking that maybe a better fix would be to not just restore the minimized group but also set it active. It feels weird to me that you click on an item and you are not getting any feedback if the item is already opened in some group. At the very minimum I think it should not only restore (if minimized) but also activate the group. Would you agree?

**Comment by djmisterjon** [comment]
@djmisterjon yes +1 for active group 👍

**Comment by bpasero** [comment]
OK, pushed that via https://github.com/microsoft/vscode/issues/79798

**Comment by bpasero** [comment]
@mjbvz good catch, this was obviously broken when I made all extensions go back to the previous solution and the "Find References" view is an extension.

I added another value to the `EditorActivation` option: `RESTORE` and now all extensions (including your webview stuff) will use that option. This option will NOT activate the group (so we preserve todays behavior), but will restore a group IF it is minimized.
